### DISCUSSION TITLE
How do I use the themes in the root 'themes' folder?

### DISCUSSION THREAD

**Discussion by crimsonnight** [mathoudebine/turing-smart-screen-python#122] [Themes]
There's only 1 landscape theme in a folder and I'm not a fan - there are some good looking themes in the root folder though, 'LandscapeMagicBlue_theme' etc. How do I use one of these themes instead of the ones in the folders? Cheers

**Comment by mathoudebine** [comment]
Hi @crimsonnight 
The files in the root folder are only examples: you would find a background file and an example of what could be done by creating a theme with this background.
They cannot be used as theme because they do not have a specific folder and the `theme.yaml` corresponding file has not been written to specify where the content would be displayed over the background.
If you want, you can implement one of these examples as a theme yourself:
- create a folder for this theme in `res/themes`
- move the corresponding background file
- copy a `theme.yaml` from another theme (like the other landscape theme)
- edit the `theme.yaml` to show sensors over the background at the right place to recreate what is shown in the example file

FYI I try to do this work when i have some free time, but as you can see it is a time-consuming task because every data needs to be positioned manually at the right place - no theme editor for now.

**Reply by crimsonnight** [reply]
Thought that might be the case - might be worth just labelling changing the filenames to include 'example' or something.
And cheers, hopefully something uploads them at some point (don't think I have the technical skills!)

**Reply by mathoudebine** [reply]
Update: i uploaded the complete LandscapeMagicBlue theme to the master branch, you can try it if you want! There are 2 circles that do not have data in it for now but they will be filled in a future version
![ThemeLandscapeMagicBlue](https://user-images.githubusercontent.com/38615348/210270851-a96fa040-8aa1-4de6-b119-6ec1b01319fc.png)

**Reply by crimsonnight** [reply]
Oh great cheers, I'll check it out! :) I managed to create my own theme in the meantime, but it's just a modded version of the included landscape theme.
